### DISCUSSION TITLE
Feature Requests: Tour

### DISCUSSION THREAD

**Discussion by krnwnadhi** [mantinedev/mantine#6739] [Feature requests]
Hi Mantine Team, i think we need a tour component like this one:  https://ant.design/components/tour 
Thank you Mantine.

**Comment by rtivital** [comment]
It is better to use a third-party library for that, for example – https://react-joyride.com/

**Comment by matheusvellone** [comment]
https://github.com/gfazioli/mantine-onboarding-tour
